EXAMPLES.md

```diff
@@ -0,0 +1,258 @@
+
+Passwordless is a two-step authentication flow that makes use of this type of connection. The **Passwordless OTP** grant is required to be enabled in your Auth0 application beforehand. Check [our guide](https://auth0.com/docs/dashboard/guides/applications/update-grant-types) to learn how to enable it.
+
+To start the flow, you request a code to be sent to the user's email or phone number. For email scenarios only, a link can be sent in place of the code.
+
+```js
+auth0.auth
+  .passwordlessWithEmail({
+    email: 'info@auth0.com',
+    send: 'link',
+  })
+  .then(console.log)
+  .catch(console.error);
+```
+
+or
+
+```js
+auth0.auth
+  .passwordlessWithSMS({
+    phoneNumber: '+5491159991000',
+  })
+  .then(console.log)
+  .catch(console.error);
+```
+
+Then, in order to complete the authentication, you must send back that received code value along with the email or phone number used:
+
+```js
+auth0.auth
+  .loginWithEmail({
+    email: 'info@auth0.com',
+    code: '123456',
+  })
+  .then(console.log)
+  .catch(console.error);
+```
+
+or
+
+```js
+auth0.auth
+  .loginWithSMS({
+    phoneNumber: '+5491159991000',
+    code: '123456',
+  })
+  .then(console.log)
+  .catch(console.error);
+```
+
+### Create user in database connection
+
+```js
+auth0.auth
+  .createUser({
+    email: 'info@auth0.com',
+    username: 'username',
+    password: 'password',
+    connection: 'myconnection',
+  })
+  .then(console.log)
+  .catch(console.error);
+```
+
+## Management API (Users)
+
+### Patch user with user_metadata
+
+```js
+auth0
+  .users('the user access_token')
+  .patchUser({id: 'user_id', metadata: {first_name: 'John', last_name: 'Doe'}})
+  .then(console.log)
+  .catch(console.error);
+```
+
+### Get full user profile
+
+```js
+auth0
+  .users('the user access_token')
+  .getUser({id: 'user_id'})
+  .then(console.log)
+  .catch(console.error);
+```
+
+For more info please check our generated [documentation](https://auth0.github.io/react-native-auth0/index.html)
+
+## Organizations
+
+[Organizations](https://auth0.com/docs/organizations) is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.
+
+Using Organizations, you can:
+
+- Represent teams, business customers, partner companies, or any logical grouping of users that should have different ways of accessing your applications, as organizations.
+- Manage their membership in a variety of ways, including user invitation.
+- Configure branded, federated login flows for each organization.
+- Implement role-based access control, such that users can have different roles when authenticating in the context of different organizations.
+- Build administration capabilities into your products, using Organizations APIs, so that those businesses can manage their own organizations.
+
+Note that Organizations is currently only available to customers on our Enterprise and Startup subscription plans.
```

Comment:
```suggestion
[Organizations](https://auth0.com/docs/organizations) is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.

Note that Organizations is currently only available to customers on our Enterprise and Startup subscription plans.
```

Reply:
As mentioned on slack, we are getting rid of some of the organizations marketing text, as we are already point to the docs above.